roles/container-engine/docker/tasks/reset.yml

```diff
@@ -78,7 +78,7 @@
   when: ansible_distribution == "Fedora" and not is_ostree
 
 - name: Docker | Remove docker repository on RedHat/CentOS/Oracle/AlmaLinux Linux
-  template:
+  file:
     name: "{{ yum_repo_dir }}/docker-ce.repo"
```

Comment:
On ansible.builtin.file module, I guess a file needs to be specified with `path`, not `name` according to https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html

Reply:
> path 
> aliases: dest, name

It seems that both path and name is ok.